bundles/org.eclipse.jface/src/org/eclipse/jface/resource/URLImageDescriptor.java

```diff
@@ -130,12 +130,6 @@ public boolean equals(Object o) {
 		return ((URLImageDescriptor) o).url.equals(this.url);
 	}
 
-	@Deprecated
-	@Override
-	public ImageData getImageData() {
-		return getImageData(getURL(url), 100, 100);
-	}
```

Comment:
@Michael5601 FYI as this was a relevant point in our discussion at https://github.com/eclipse-platform/eclipse.platform.ui/pull/2593. As before that change this implementation now just behaves as if `getImageData(100)` is called and therefore as the default implementation in the super-class. It can therefore just be removed.